internal/cli/project_inspect.go

```diff
@@ -0,0 +1,327 @@
+	if project.DataSourcePoll != nil {
+		projectPollEnabled = project.DataSourcePoll.Enabled
+		projectPollInterval = project.DataSourcePoll.Interval
+	}
+
+	var appPollEnabled bool
+	var appPollInterval string
+	if project.StatusReportPoll != nil {
+		appPollEnabled = project.StatusReportPoll.Enabled
+		appPollInterval = project.StatusReportPoll.Interval
+	}
+
+	if c.flagJson {
+		projectHeaders := []string{
+			"Project", "Applications", "Workspaces", "Remote Enabled", "Data Source", "Project Poll Enabled",
+			"Project Poll Interval", "App Status Poll Enabled", "App Status Poll Interval",
+		}
+
+		projectTbl := terminal.NewTable(projectHeaders...)
+
+		statusColor := ""
+		columns := []string{
+			project.Name,
+			strings.Join(appNames, "\n"),
+			strings.Join(workspaceNames, "\n"),
+			strconv.FormatBool(project.RemoteEnabled),
+			dataSource,
+			strconv.FormatBool(projectPollEnabled),
+			projectPollInterval,
+			strconv.FormatBool(appPollEnabled),
+			appPollInterval,
+		}
+
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+
+		err = c.outputProjectJson(projectTbl)
+		if err != nil {
+			return err
+		}
+	} else {
+		projectHeaders := []string{
+			"Project Option", "Value",
+		}
+
+		projectTbl := terminal.NewTable(projectHeaders...)
+		statusColor := ""
+
+		// Show project info in a flat list where each project option is its
+		// own row
+		columns := []string{
+			"Project Name",
+			project.Name,
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+		columns = []string{
+			"Applications",
+			strings.Join(appNames, "\n"),
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+		columns = []string{
+			"Workspaces",
+			strings.Join(workspaceNames, "\n"),
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+		columns = []string{
+			"Remote Enabled",
+			strconv.FormatBool(project.RemoteEnabled),
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+		columns = []string{
+			"Data Source",
+			dataSource,
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+		columns = []string{
+			"Project Poll Enabled",
+			strconv.FormatBool(projectPollEnabled),
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+		columns = []string{
+			"Project Poll Interval",
+			projectPollInterval,
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+		columns = []string{
+			"App Status Poll Enabled",
+			strconv.FormatBool(appPollEnabled),
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+		columns = []string{
+			"App Status Poll Interval",
+			appPollInterval,
+		}
+		projectTbl.Rich(
+			columns,
+			[]string{
+				statusColor,
+			},
+		)
+
+		c.ui.Table(projectTbl)
+	}
+
+	return nil
+}
+
+func (c *ProjectInspectCommand) outputProjectJson(t *terminal.Table) error {
```

Comment:
I sort of want to make this something on terminal so that we can use it from all the commands. Thoughts? Or maybe for now, just make it a toplevel function rather than something on projectInspectCommand?

Reply:
I want this too. I almost made it a top level function since this function is mostly identical from the one I wrote for the Status command. The only difference is for applications and workspaces I'm splitting on `\n` to make an array. But maybe we're safe to say if we detect any `\n` in a value, it's multiple values and should be an array? I could see this coming in handy a bunch though as we add more CLI helpers.

Reply:
Done!